app/javascript/widget/store/modules/contacts.js

```diff
@@ -38,7 +38,11 @@ export const actions = {
         dispatch('conversation/fetchOldConversations', {}, { root: true });
       }
     } catch (error) {
-      // Ignore error
+      const {
+        response: { data },
+      } = error;
+
+      dispatchWindowEvent({ eventName: 'chatwoot:error', data: data.error });
```

Comment:
Please make a constant file for chatwoot SDK events. move `chatwoot:ready` and `chatwoot:error` to that.